.release-notes/3745.md

```diff
@@ -0,0 +1,5 @@
+## Update to LLVM 12.0.1
+
+Updated the vendored LLVM submodule to tag `llvmorg-12.0.1`, and updated the build system, code generation, and JIT code to work with LLVM 12.
+
+Disabled 32-bit ARM cross-compilation tests pending fixes for 32-bit ARM and M1.
```

Comment:
I dont think this is release note worthy.

Are we basically saying though that we are dropping 32-bit ARM support? I believe so, so we should so that.

Reply:
updated